manuscript.md

```diff
@@ -55,6 +55,10 @@
 ## How could predictions for individual species, such as those used by IPBES/IUCN, be improved by considering ecological interactions?
 
 ## What is the spatial scale suitable for the prediction of species interactions?
```

Comment:
The markdown convention is to have a space in between a title and the rest of the text